### PR TITLE
Update the shapefile export status bar with each feed in a deploy…

### PR DESCRIPTION
**Pull request by philip-cline**
…ment export

### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Updates the status bar w/ each feed GisExportJob that finishes.

### CODE REVIEW

File: src/main/java/com/conveyal/datatools/manager/jobs/DeploymentGisExportJob.java [hunk 1]
```diff
@@ -29,15 +29,21 @@ public DeploymentGisExportJob(GisExportJob.ExportType exportType, Deployment dep
         try {
             File outDir = setupGisExport();
 
+            int feedVersionsSize = feedVersions.size();
+            int currentFeedVersion = 1;
             for (SummarizedFeedVersion feedVersion: feedVersions) {
+                int percentComplete = (int) (currentFeedVersion * 100.0f) / feedVersionsSize;
```
Comment (miles-grant-ibigroup) [inline src/main/java/com/conveyal/datatools/manager/jobs/DeploymentGisExportJob.java]: Are there any java math util we can use to avoid this cast?

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks and works great! Part of me thinks there might be a way to avoid the dreaded `currentFeedVersion++` but nothing off the top of my head...

File: src/main/java/com/conveyal/datatools/manager/jobs/DeploymentGisExportJob.java (hunk 1 above)
Comment (philip-cline) [reply src/main/java/com/conveyal/datatools/manager/jobs/DeploymentGisExportJob.java]: How you like these apples: [34cd337](https://github.com/ibi-group/datatools-server/pull/471/commits/34cd337257dbab288e39a39a044f2f41cca2b871)

Comment (miles-grant-ibigroup) [reply src/main/java/com/conveyal/datatools/manager/jobs/DeploymentGisExportJob.java]: Delicious!

**Comment by philip-cline** [comment]
For including an index in the forEach loop, my understanding was that this couldn't be done without creating an index manually ([ref](https://stackoverflow.com/questions/477550/is-there-a-way-to-access-an-iteration-counter-in-javas-for-each-loop)) but perhaps we could switch to an iterator if it's worth it.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Comment by binh-dam-ibigroup** [comment]
> For including an index in the forEach loop, my understanding was that this couldn't be done without creating an index manually ([ref](https://stackoverflow.com/questions/477550/is-there-a-way-to-access-an-iteration-counter-in-javas-for-each-loop)) but perhaps we could switch to an iterator if it's worth it.

The code is fine as written.

**Pull request merged**